### PR TITLE
Add an option named ignore-source-changes

### PR DESCRIPTION
**Pull request by arnaudmorin**
This add a new option ignore-source-changes which is false by default.
This option can be used to force building a package even if there is
some changes in the source code.
It basically add -i.* to dpkg-buildpackage, which is then given to
dpkg-source.

Signed-off-by: Arnaud Morin <arnaud.morin@corp.ovh.com>

### CODE REVIEW

File: stdeb/util.py [hunk 1]
```diff
@@ -109,6 +109,9 @@ def check_call(*popenargs, **kwargs):
      'Allow installing into /some/random/virtualenv-path'),
     ('sign-results',None,
      'Use gpg to sign the resulting .dsc and .changes file'),
+    ('ignore-source-changes',None,
+     'Ignore all changes on source when building source package (add -i.* '
+     'option to dpkg-source'),
```
Comment (dirk-thomas) [inline stdeb/util.py]: Nitpick: please add the missing closing parenthesis in the help message.

**Comment by dirk-thomas** [comment]
The new option should also be documented in the README.

**Comment by dirk-thomas** [comment]
@arnaudmorin Friendly ping.

File: stdeb/util.py (hunk 1 above)
Comment (arnaudmorin) [reply stdeb/util.py]: done

**Comment by arnaudmorin** [comment]
> The new option should also be documented in the README.

done.

sorry for late answer :p

File: stdeb/util.py [hunk 2]
```diff
@@ -1434,9 +1439,16 @@ def build_dsc(debinfo,
...
     else:
-        args = ('-uc','-us')
+        sign_dsc_args = ('-uc','-us')
+
+    if ignore_source_changes:
+        ignore_source_changes_args = ('-i.*',)
+    else:
+        ignore_source_changes_args = ()
+
+    args = sign_dsc_args + ignore_source_changes_args
```
Comment (dirk-thomas) [inline stdeb/util.py:1451]: How about consolidating this logic to:

```
args = []
if not sign_dsc:
    args += ['-uc','-us']
if ignore_source_changes:
    args.append('-i.*')

Comment (dirk-thomas) [reply stdeb/util.py:1451]: @arnaudmorin Friendly ping.

Comment (arnaudmorin) [reply stdeb/util.py]: done

**Comment by dirk-thomas** [comment]
Thank you for the contribution an apologies for the slow response.

**Pull request merged**